app/views/groups/manage_members.html.erb

```diff
@@ -92,7 +92,7 @@
               <thead>
                 <tr>
                   <th><%= User.human_attribute_name(:name) %></th>
-                  <th><%= t('.request_made_at') %></th>
+                  <th><%= I18n.t('.request_made_at') %></th>
```

Comment:
I know-- this shouldn't be needed, but it wouldn't work without it. And even wierder, it was resolving the path to the root... hence the entry in the en.yml.